src/App.tsx

```diff
@@ -1,10 +1,74 @@
-import React from 'react';
+import React, { FC } from 'react';
 import './App.css';
 
-const App = () => (
-  <div className="App">
-    <h1>Redux list of todos</h1>
-  </div>
-);
+import { connect } from 'react-redux';
+import {
+  setLoading, setError, getLoading, getError, Storage, setNewTodos, getTodos,
+} from './store';
 
-export default App;
+import { loadTodos, loadUsers } from './utils/utils';
+import { TodoWithUser } from './utils/types';
+import { TodoList } from './components/TodoList';
+
+interface Props {
+  isLoading: boolean;
+  isError: string;
+  todos: TodoWithUser[];
+  setLoading: (value: boolean) => void;
+  setError: (value: string) => void;
+  setNewTodos: (value: TodoWithUser[]) => void;
+}
+
+const App: FC<Props> = (props) => {
+  const {isLoading, isError, todos, setLoading, setError, setNewTodos} = props;
```

Comment:
линтер присутствует?
```suggestion
  const {
    isLoading,
    isError,
    todos,
    setLoading,
    setError,
    setNewTodos,
  } = props;
```